components/dashboard/src/teams/Members.tsx

```diff
@@ -257,16 +210,17 @@ export default function MembersPage() {
                                             m.userId === user?.id
                                                 ? [
                                                       {
-                                                          title: leaveTeamEnabled
+                                                          title: org.data?.isOwner
                                                               ? "Leave Organization"
                                                               : "Remaining owner",
-                                                          customFontStyle: leaveTeamEnabled
+                                                          customFontStyle: org.data?.isOwner
                                                               ? "text-red-600 dark:text-red-400 hover:text-red-800 dark:hover:text-red-300"
                                                               : "text-gray-400 dark:text-gray-200",
-                                                          onClick: () => leaveTeamEnabled && removeTeamMember(m.userId),
+                                                          onClick: () =>
+                                                              org.data?.isOwner && removeTeamMember(m.userId),
```

Comment:
@svenefftinge this read like you would not be able to leave an org as a non-owner.
also, every owner would see "Remaining owner".

Before the change, this would only be shown for the last owner of an org.

Reply:
oh, yes. It should be that you cannot leave if you are the last owner.

Reply:
I've filed an issue https://github.com/gitpod-io/gitpod/issues/16753